### PR TITLE
[Cloud Sync] Merge remote-tracking branch 'origin/master' 2021-08-02

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Merge master into cloud branch for cloud release

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by pablovelezvidal — APPROVED** [review]
hey @isacikgoz heads up that Maria Nuñez is on vacation this week. Not sure if this needs to be validated by a lead or otherwise you can change the reviewer from Maria to @AGMETEOR

File: i18n/nl.json [hunk 1]
```diff
@@ -4217,7 +4217,7 @@
   "status_dropdown.set_online": "Online",
   "status_dropdown.set_ooo": "Out Of Office",
   "status_dropdown.set_ooo.extra": "Automatische Antwoorden zijn ingeschakeld",
-  "status_dropdown.set_your_status": "Stel je status in",
+  "status_dropdown.set_your_status": "Status",
```
Comment (AGMETEOR) [inline i18n/nl.json:4220]: Wondering if this is an intentional change?

File: e2e/package-lock.json [hunk 2]
```diff
@@ -21578,17 +21578,6 @@
         "mochawesome-report-generator": "^5.2.0",
         "strip-ansi": "^6.0.0",
         "uuid": "^8.3.2"
-      },
```
Comment (AGMETEOR) [inline e2e/package-lock.json:21581]: Wondering if this is an intentional change

File: i18n/nl.json (hunk 1 above)
Comment (isacikgoz) [reply i18n/nl.json]: I changed the base, this change was due to that. Can you check again?

**Comment by amyblais** [comment]
@AGMETEOR Kind reminder for PR review.

**Review by jwilander — APPROVED** [review]

**Pull request merged**